ShoppingList/Articles.cs

```diff
@@ -11,26 +12,24 @@ public class Articles
         public string NameOfThing;
         public int amount;
         public decimal price;
-        private decimal realPrice;
+        public List<Articles> article = new();
         public Articles() // Construct 
         {
             NameOfThing = "Not Define";
             amount = 0;
-            price = 0;
-            if(amount > 1)
-            {
-                realPrice = this.RealPrice();
-            }
-            else
-            {
-                realPrice = price;
-            }
+            price =  price * amount;
         }
-        private decimal RealPrice()
+        public virtual void CreateANewArticle()
```

Comment:
Why is this function virtual, also I would change this function so that it accepts those parameters from outside instead of calling WriteLine or ReadLine inside.